README.md

```diff
@@ -1,2 +1,26 @@
-# kodo
-##### testing
+**<h1>Command line utilities using cobra and Go</h1>**
+>A go program to develop command line utilities with commands, subcommands, aliase etc.
+
+**Dependencies**
```

Comment:
Question I would ask is:
* Does a contributor need to know this ?  ( already present in go.mod )
* Does a user needs to know this ?

Reply:
You are right, that go.mod already have dependencies list. 
But for a new go learner, it might take time to figure out to refer mod file for dependencies (just a thought). But I can remove if needed in next PR.

Reply:
Push your changes to this branch/PR please.

Reply:
@sbose Yes!!